crates/ruff_linter/resources/test/fixtures/airflow/AIR302_context.py

```diff
@@ -0,0 +1,19 @@
+from airflow.decorators import task
+
+@task
+def print_config(**context):
+    # This should not throw an error as logical_date is part of airflow context.
+    logical_date = context["logical_date"]
+
```

Comment:
@sunank200 I just think of there're other ways to access the context value like https://airflow.apache.org/docs/apache-airflow/stable/templates-ref.html#accessing-airflow-context-variables-from-taskflow-tasks. Do you think we should check it as well?

Reply:
I have added logic for other ways to access context value as well. It is part of tests.